### PR TITLE
Rustfmt

### PR DESCRIPTION
**Pull request by Vurich**
Submitting this PR pleases me immensely

### CODE REVIEW

File: example/src/lib.rs [hunk 1]
```diff
@@ -43,22 +43,22 @@ where
...
+        let mut multiaddr = address
+            .parse::<Multiaddr>()
+            .expect("failed to parse hard-coded multiaddr");
 
-		let ipfs_component = multiaddr.pop().expect("hard-coded multiaddr is empty");
-		let public_key = match ipfs_component {
-			multiaddr::AddrComponent::IPFS(key) => key,
-			_ => panic!("hard-coded multiaddr didn't end with /ipfs/"),
-		};
+        let ipfs_component = multiaddr.pop().expect("hard-coded multiaddr is empty");
```
Comment (rphmeier) [inline example/src/lib.rs]: killed tabs in this file

**Comment by tomaka** [comment]
What about adding a style check to `.travis.yml`?

File: libp2p-dns/src/lib.rs [hunk 2]
```diff
@@ -62,221 +62,221 @@ use tokio_dns::{CpuPoolResolver, Resolver};
...
+        trace!(target: "libp2p-dns", "Created a CpuPoolResolver");
 
-		DnsConfig {
-			inner,
-			resolver: CpuPoolResolver::new(num_threads),
-		}
-	}
+        DnsConfig {
+            inner,
+            resolver: CpuPoolResolver::new(num_threads),
```
Comment (rphmeier) [inline libp2p-dns/src/lib.rs]: and here

File: libp2p-peerstore/src/memory_peerstore.rs [hunk 3]
```diff
@@ -34,80 +34,83 @@ use std::vec::IntoIter as VecIntoIter;
...
+        OwningRefMut::new(lock)
+            .try_map_mut(|n| n.get_mut(peer_id).ok_or(()))
+            .ok()
+            .map(MemoryPeerstoreAccess)
+    }
+
+    fn peer_or_create(self, peer_id: &PeerId) -> Self::PeerAccess {
+        let lock = self.store.lock().unwrap();
+        let r = OwningRefMut::new(lock)
+            .map_mut(|n| n.entry(peer_id.clone()).or_insert_with(|| PeerInfo::new()));
```
Comment (rphmeier) [inline libp2p-peerstore/src/memory_peerstore.rs]: killed tabs

File: libp2p-peerstore/src/peer_info.rs [hunk 4]
```diff
@@ -37,130 +37,133 @@ use std::time::{Duration, SystemTime, UNIX_EPOCH};
...
 impl PeerInfo {
-	/// Builds a new empty `PeerInfo`.
-	#[inline]
-	pub fn new() -> PeerInfo {
-		PeerInfo { addrs: vec![] }
-	}
+    /// Builds a new empty `PeerInfo`.
+    #[inline]
+    pub fn new() -> PeerInfo {
+        PeerInfo { addrs: vec![] }
```
Comment (rphmeier) [inline libp2p-peerstore/src/peer_info.rs]: killed tabs

**Comment by rphmeier** [comment]
I think it's formatting as spaces instead of tabs

**Comment by Vurich** [comment]
@rphmeier Yes, because it's rustfmt with default styling. The `rustfmt.toml` we had before didn't represent RFC styling or parity styling, so we decided to just use default styling since parity styling is not possible to express in a rustfmt config (as we decided before).

**Comment by Vurich** [comment]
Closed by #137

**Pull request closed without merging**